content/cookbook/tokens/close-token-accounts.md

```diff
@@ -0,0 +1,80 @@
+---
+title: How to close token accounts
+sidebarSortOrder: 10
+description:
+  "Learn how to close Solana token accounts, including Wrapped SOL and
```

Comment:
-> "Learn how to close token accounts on Solana, including cases such as Wrapped SOL and empty accounts."